.github/ISSUE_TEMPLATE/Django Feature.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Django Feature Request
+about: Suggest an idea for this project
+title: ""
+labels: 'Django feature'
+assignees: ''
+
+---
+
+## Django Feature Request
+
+This feature request is for features requeted to the backend (Django) of Auto-DL.
```

Comment:
The spelling of "related" is wrong.  Also put the statement in comments.

Reply:
Corrected the spelling and also added to comments